### PR TITLE
OpenStack: remove region from install-config

### PR DESCRIPTION
**Pull request by Fedosin**
This parameter is not used by the installer and completely ignored during the installation, so it just confuses users and should be removed.

The real region value is taken from clouds.yaml file.

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
/hold

Removing fields is backwards incompatible change, and we need cannot make such a change.

**Comment by smarterclayton** [comment]
/hold

Agree, default position is you may never remove fields, just document them unused and deprecated.  Changing the godoc is sufficient (if we don’t publish a go schema for this)

**Comment by abhinavdahiya** [comment]
> https://github.com/openshift/installer/pull/2332/files#diff-26eeddc74b12e6afb9e2932066a9b49f

@Fedosin can you keep the field and update the godoc to say, this is field is unused and deprecated... and you can keep the rest of the change that just ignores this field.

That should be enough to move ahead on this.

**Comment by tomassedovic** [comment]
@Fedosin having followed the discussion on field removal I agree with Clayton and Abhinav. Please keep the region field in the `Platform` struct & update the doc. I'll review it later -- it looks good to me otherwise.

**Comment by Fedosin** [comment]
@tomassedovic @abhinavdahiya  @smarterclayton ack! I will update the patch soon

**Comment by Fedosin** [comment]
/hold cancel

returned Region to the Platform

**Comment by tomassedovic** [comment]
/retest

Libvirt package mirror flakes looks like.

File: pkg/types/openstack/platform.go [hunk 1]
```diff
@@ -3,7 +3,8 @@ package openstack
 // Platform stores all the global configuration that all
 // machinesets use.
 type Platform struct {
-	// Region specifies the OpenStack region where the cluster will be created.
+	// Region
```
Comment (abhinavdahiya) [inline pkg/types/openstack/platform.go]: can we keep the field comment on the region.. i don't see the reason of removing it..

Comment (abhinavdahiya) [reply pkg/types/openstack/platform.go]: also update the https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md

Comment (Fedosin) [reply pkg/types/openstack/platform.go]: Ack. Will be done

**Comment by abhinavdahiya** [comment]
/approve

cc @sdodson for approval.

**Comment by tomassedovic** [comment]
LGTM from OpenStack engineering side. Waiting on @sdodson before making it official so it doesn't merge before he's had his say.

**Comment by smarterclayton** [comment]
doc looks fine to me

**Pull request merged**